Contentful.AspNetCore/Contentful.AspNetCore.csproj

```diff
@@ -0,0 +1,48 @@
+﻿<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" />
+  <PropertyGroup>
+    <Description>Official .NET SDK for the Contentful Content Delivery and management API for ASP.NET core.</Description>
```

Comment:
Sorry, a bit of nit picking: If we write `Content Delivery` should we also write `Management` (Capital letters?)